docs/dev_guide/pod_lifecycle.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This is definitely not a part of the dev guide, but rather details around the internal of the product. It should be a part of the design guide.